packages/node_modules/@webex/plugin-meetings/README.md

```diff
@@ -66,6 +66,170 @@ return spark.meetings
+    events: true // with events
+  },
+  media: { // can add media or exclude
+    ...
+  }
+});
+meeting.getStats().getScreen().onData((filtered) => {
+  console.log(filtered);
+});
+// afterwards, to have the same instance you may just call
+meeting.getStats()...
+```
+
+##### Adding Configurations On The Fly
+```javascript
+// you can always add additional configurations later on based on events, or some timing, using the builder above
+// say you created the object above with the options method, and you decided you wanted to add an aggregator to it
+meeting
+.getStats()
+.getScreen()
+.withAggregator();
+// and now you have an aggregator instance to work with on that screen stats object
+meeting.stats.screen.aggregator...
+```
+
+##### Aggregator
+```javascript
+// note: using the aggregator is probably not necessary unless the WebRTC spec changes, or browsers change behavior and no longer automatically
+// summarize their data
+meeting
+.getStats() // get the stats instance on the meeting
+.getMedia() // get the media peer connection related stats, can also do getScreen()
+.getAggregator() // get the aggregator instance on that media stats
+.getSummary() // get the last found summary object
+```
+
+##### Events
+```javascript
+meeting
+.getStats() // get the stats instance on the meeting
+.getMedia() // get the media peer connection related stats, can also do getScreen()
+.getEvents() // get the events instance on that media stats
+.on('stats:update', (stats) => { // listen to the stats update event that comes through here
+  // do something with the stats event, see Events below
+});
+```
+
+##### History
+```javascript
+meeting
+.getStats() // get the stats instance on the meeting
+.getMedia() // get the media peer connection related stats, can also do getScreen()
+.getHistory() // get the history instance on that media stats
+.getHistory() // get the actual array of history stats objects
+```
+
+##### Interval
+```javascript
+meeting
+.getStats() // get the stats instance on the meeting
+.getMedia() // get the media peer connection related stats, can also do getScreen()
+.getInterval() // get the interval instance on that media stats
+.getInterval() // get the actual interval object that was last saved
+```
+
+##### Putting It All Together
+```javascript
+// create
+meeting
+.getStats()
+.withMedia()
+.build();
+meeting
+.getStats()
+.getMedia()
+.withId()
+.withFilter(meeting.mediaPeerConnection)
+.withEvents()
+.withHistory()
+.withInterval()
+.withAggregator()
+.build();
+meeting
+.getStats()
+.withScreen()
+.build();
+meeting
+.getStats()
+.getScreen()
+.withId()
+.withStream(meeting.screenPeerConnection)
+// do stuff
+meeting
+.getStats()
+.getMedia()
+.getStream()
+.on('data', (data) => {
+  console.log(data);
+});
+meeting
+.getStats()
+.getMedia()
+.onData((filtered) => {
+  console.log(filtered);
+});
+meeting
+.getStats()
+.getMedia()
+.getEvents()
+.on('stats:update', (stats) => {
+  console.log(stats);
+});
```

Comment:
That is knarly. I haven't seen much like it in the wild but I'm open to it. @mattnorris? Great README overall.

Reply:
ya that was due to the stateless spark plugin - it doesn't trigger on just the meeting object